process.c

```diff
@@ -3058,10 +3082,22 @@ NORETURN(static VALUE f_exec(int c, const VALUE *a, VALUE _));
  *  - A 2-element array containing the path to an executable
  *    and the string to be used as the name of the executing process.
  *
- *  Ruby invokes the executable directly, with no shell and no shell expansion.
+ *  Example:
+ *
+ *    exec('/usr/bin/date')
+ *
+ *  Output:
+ *
+ *    Sat Aug 26 09:38:00 AM CDT 2023
+ *
+ *  Ruby invokes the executable directly, with no shell and no shell expansion:
+ *
+ *    [Needs example.]
```

Comment:
An example is that this raises a Ruby error:

```ruby
exec('doesnt_exist') # => Raises Errno::ENOENT (No such file or directory - doesnt_exist)
```

While adding a meta character invokes the shell:

```ruby
exec('doesnt_exist && echo ok') # => sh: doesnt_exist: command not found
```